packages/react/src/components/Notification/Notification-story.js

```diff
@@ -26,7 +26,11 @@ const notificationProps = () => ({
   lowContrast: boolean('Use low contrast variant (lowContrast)', false),
   role: text('ARIA role (role)', 'alert'),
   title: text('Title (title)', 'Notification title'),
-  subtitle: text('Subtitle (subtitle)', 'Subtitle text goes here.'),
+  subtitle: (
+    <span>
+      Subtitle text goes here. <a href="#example">Example link</a>
```

Comment:
I'm not sure if this will fix it but the `a` tag here is causing a color contrast DAP violation in low contrast mode, would switching it to the `Link` component fix it? 
```suggestion
      Subtitle text goes here. <Link href="#example">Example link</Link>
```

Reply:
Ignore that! It doesn't resolve the violation.

Reply:
Thanks @abbeyhrt! I was thinking in this case that a user may be likely to just pass in a plain anchor element instead of an actual Link component, since the prop is a `node` type.

About the colors... I went with `$inverse-link` because the token seemed appropriate, but I can definitely adjust to something lighter? 🤔 I'm also unsure what the color of the link should be when you hover/focus on it... as it is, I can't necessarily leave it as-is because the component is inverted so any anchor/`Link` passed it will be even more unreadable (like blue60 by default 😅 )

What do you think about the colors though? Lighter like Blue30 or something like that?

Reply:
Ah that makes sense! I'm finding the color contrast violation only exists in low contrast mode in the white and g10 themes, so I would think we would need a darker value? But since it takes on the theme value, it can't be too dark or it causes a violation in the default mode 🤦‍♀ .  I'm thinking we would need to do something special for links when low contrast mode is on.  

For the hover, maybe we could follow the pattern 'Link' has? Toggling the underline on hover.